driver/utils/type_info.h

```diff
@@ -1813,6 +1813,9 @@ namespace value_manip {
         template <typename SourceType>
         struct from_value {
             static inline SQLRETURN convert(const SourceType & src, BindingInfo & dest) {
+                if (dest.indicator && dest.indicator != dest.value_size)
+                    *dest.indicator = 0; // Value is not null here...
```

Comment:
Not quite sure what `indicator` means here, could you please add some comments to `BindingInfo`? Please also specify what does it mean when `indicator` value is `0`.

Reply:
Modified the comments.